test/docs/variable_mapping_test.rb

```diff
@@ -35,21 +35,12 @@ def create_model(ctx, current_user:, **)
         #~io-steps end
       end
       #:io-ary-hash end
-
-      signal, (ctx, flow_options) = Activity::TaskWrap.invoke(A::Memo::Create, [{params: {id: 1}}, {}])
-      signal.inspect.must_equal %{#<Trailblazer::Activity::End semantic=:success>}
-      ctx.inspect.must_equal %{{:params=>{:id=>1}, :current_user=>\"User 1\", :model=>\"User 1\"}}
-=begin
-#:io-call
-ctx = {params: {id: 1}}
-
-signal, (ctx, flow_options) = Activity::TaskWrap.invoke(A::Memo::Create, [ctx, {}])
-
-ctx #=> {:params=>{:id=>1}, :current_user=>#<User ..>, :model=>#<Memo ..>}}
-#:io-call end
-=end
     end
 
+    signal, (ctx, flow_options) = Activity::TaskWrap.invoke(A::Memo::Create, [{params: {id: 1}}, {}])
```

Comment:
Or instead, how about we wrap this in between `#:io-call`  & `#:io-call end`

Reply:
I wont have access to the _